test/kpod_create.bats

```diff
@@ -2,23 +2,24 @@
 
 load helpers
 
+function setup() {
+    copy_images
+}
+
 function teardown() {
     cleanup_test
 }
 
 ALPINE="docker.io/library/alpine:latest"
 
 @test "create a container based on local image" {
-    run ${KPOD_BINARY} ${KPOD_OPTIONS} pull docker.io/library/busybox:latest
-    echo "$output"
-    [ "$status" -eq 0 ]
     run ${KPOD_BINARY} ${KPOD_OPTIONS} create docker.io/library/busybox:latest ls
```

Comment:
Should this be referring to the image as `$BB`?

Reply:
fixed